service/extensions/config.go

```diff
@@ -0,0 +1,22 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extensions // import "go.opentelemetry.io/collector/service/extensions"
+
+import (
+	"go.opentelemetry.io/collector/config"
+)
+
+// Config represents the ordered list of extensions configured for the service.
+type Config []config.ComponentID
```

Comment:
Should we have a struct in case we ever want to add more fields?

```suggestion
// Config defines the Extensions configuration
type Config struct {
  // Extensions configured for the service
  Extensions []config.ComponentID
}
```

Also, does the extension order really matter? We do start them in that order, but maybe we should not mention that in the docs, so that we are free to change that order (e.g. start extensions in parallel) if we ever want to.

Reply:
>Also, does the extension order really matter? We do start them in that order, but maybe we should not mention that in the docs, so that we are free to change that order (e.g. start extensions in parallel) if we ever want to.

Extensions may depend on other extensions (you can use GetExtensions() to access another extension), so the start order can be important (I don't we have an actual use case where it matters, but we may have in the future).

Reply:
That's a good point, we should keep the order mention then.

Reply:
Thanks for the discussion, seems to be resolved.